x/ccv/provider/keeper/proposal.go

```diff
@@ -360,76 +320,21 @@ func (k Keeper) GetPendingConsumerAdditionProp(ctx sdk.Context, spawnTime time.T
 	return prop, true
 }
 
-// BeginBlockInit iterates over the pending consumer addition proposals in order, and creates
-// clients for props in which the spawn time has passed. Executed proposals are deleted.
-//
-// See: https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/methods.md#ccv-pcf-bblock-init1
-// Spec tag:[CCV-PCF-BBLOCK-INIT.1]
+// BeginBlockInit iterates over the initialized consumers chains and creates clients for chains
+// in which the spawn time has passed
 func (k Keeper) BeginBlockInit(ctx sdk.Context) {
-	propsToExecute := k.GetConsumerAdditionPropsToExecute(ctx)
```

Comment:
Just a remincer, GetConsumerAdditionPropsToExecute seems unused now and should be removed